docs/docs/about/testimonials-quotes.mdx

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 2
+---
+
+# Testimonials
+
+It's easy to say that we have a great product, but it's even better when our users say it for us. 
+Here are some of the testimonials we've received.
+
+If you would like to share your feedback with us, please [start a new discussion](https://github.com/pnp/cli-microsoft365/discussions/new?category=kind-words).
+
+## WM REPLY
+
+<div className='row testimonial'>
+  <div className='col testimonial-image'>
+    <img src={require('!!url-loader!../images/testimonials-companies-logo/wm-reply.png').default} alt='Voitanos' />
```

Comment:
Is there a reason why we use this url-loader instead of the out-of-the-box file referencing using `@site`?
https://docusaurus.io/docs/static-assets#in-jsx

Reply:
this is how I found on web and worked 😜 So there isn't any specific reason except the fact I thought I should be done this way 🙂 .
Will refactor. thanks for the comment 👍

Reply:
@milanholemans I tired to refactor it but with no luck. I tried different approaches with import or using it in the `src` attr but also did not work 🤔. Could you guide me over this?